### PR TITLE
Update all text to specify the SHA-256 hash  function rather than SHAKE-256

### PR DESCRIPTION
**Pull request by Wind4Greg**
This PR addresses issue https://github.com/w3c/vc-di-bbs/issues/118 on switching to using the SHA-256 variant of BBS, using SHA-256 for hashes, and using SHA-256 for HMAC computations.

This PR does not update the test vectors. They will be updated in a separate PR.

***
<a href="https://pr-preview.s3.amazonaws.com/Wind4Greg/vc-di-bbs/pull/128.html" title="Last updated on Feb 5, 2024, 11:29 PM UTC (488c0fc)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-di-bbs/128/98b6dd1...Wind4Greg:488c0fc.html" title="Last updated on Feb 5, 2024, 11:29 PM UTC (488c0fc)">Diff</a>

### CODE REVIEW

**Review by dlongley — APPROVED** [review]

**Review by msporny — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -734,7 +734,7 @@ <h4>createDisclosureData</h4>
 <a href="https://www.w3.org/TR/vc-di-ecdsa/#hashmandatorynquads">Section 3.3.17
 hashMandatoryNQuads</a> of the [[DI-ECDSA]] specification, passing the values
 from the map
-<var>groups.mandatory.matching</var> and utilizing the SHAKE-256 algorithm.
+<var>groups.mandatory.matching</var> and utilizing the SHA-256 algorithm.
```
Comment (TallTed) [inline index.html:737]:
```suggestion
<var>groups.mandatory.matching</var> and using the SHA-256 algorithm.
```

File: index.html [hunk 2]
```diff
@@ -1116,15 +1116,15 @@ <h4>Base Proof Hashing (bbs-2023)</h4>
...
+i.e., SHA-256. Note: This step can be performed in parallel;
 it only needs to be completed before this algorithm terminates, as the result is
 part of the return value.
             </li>
             <li>
 Initialize `mandatoryHash` to the result of calling the the algorithm in
 <a href="https://www.w3.org/TR/vc-di-ecdsa/#hashmandatorynquads">Section 3.3.17
 hashMandatoryNQuads</a> of the [[DI-ECDSA]] specification, passing
-<var>transformedDocument</var>.`mandatory` and utilizing the SHAKE-256
+<var>transformedDocument</var>.`mandatory` and utilizing the SHA-256
```
Comment (TallTed) [inline index.html:1127]:
```suggestion
<var>transformedDocument</var>.`mandatory` and using the SHA-256
```

**Review by TallTed — CHANGES_REQUESTED** [review]

**Comment by TallTed** [comment]
Probably better to resolve merge conflicts before I re-review

**Comment by msporny** [comment]
> Probably better to resolve merge conflicts before I re-review

Ready for you to re-review @TallTed

**Review by TallTed — APPROVED** [review]

**Comment by Wind4Greg** [comment]
Normative multiple reviews, no changes requested, no objections, merging.

**Pull request merged**